nativescript-background-http/background-http.ios.ts

```diff
@@ -162,7 +184,7 @@ class Session implements common.Session {
 }
 
 class Task extends Observable implements common.Task {
-	private static _tasks = new WeakMap<NSURLSessionTask, Task>();
```

Comment:
@PanayotCankov ```If the users add event listeners on the task but loose reference of the Task GC will collect the task and the NSURLSessionTask, and events will stop fire. Is this the problem?``` Yes, that sounds like the problem we were having, the task were collected, but only when not debugging! (like step by step debug), If we were debugging (step by step) then it would be fine, task would be there....

I am very new to this "pull requests" stuff, I will check how to modify this pull request and try to apply your recommended code as suggested.